Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -292,8 +292,20 @@ void addToGx2fSums(BoundMatrix& aMatrix, BoundVector& bVector, double& chi2sum,
   }
 }
 
+/// calculateDeltaParams Function
+/// This function calculates the delta parameters for a given aMatrix and
+/// bVector, depending on the number of degrees of freedom of the system, by
+/// solving the equation
+///  [a] * delta = b
+///
+/// @param aMatrix The matrix containing the coefficients of the linear system.
+/// @param bVector The vector containing the right-hand side values of the linear system.
+/// @param ndfSystem The number of degrees of freedom, determining the size of the submatrix and subvector to be solved.
+///
+/// @return deltaParams The calculated delta parameters.
 BoundVector calculateDeltaParams(const BoundMatrix& aMatrix,
```

Comment:
I this a free function which goes into `Acts::Experimental`? I think it would be good to have another namespace to collect these helpers.

Reply:
Not 100% related to this PR, but some gx2f-helpers might be a good idea :)

Reply:
Right but that should definitely be done before we move it out of `Experimental`